Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingFeatureProcessor.cpp

```diff
@@ -222,12 +222,12 @@ namespace AZ
                 meshInfo.m_bufferStartIndex = m_meshBufferIndices.AddEntry(
```

Comment:
You probably need a `meshInfo` and `materialInfo` for each device here, since the BindlesReadIndices can be different across devices